pkg/controller/operconfig/pod_controller.go

```diff
@@ -42,6 +51,14 @@ func (r *ReconcilePods) Reconcile(request reconcile.Request) (reconcile.Result,
 	}
 
 	log.Printf("Reconciling update to %s/%s\n", request.Namespace, request.Name)
+	// Trigger an update to the operconfig reconciliation loop
+	r.eventChan <- event.GenericEvent{
+		// Hard coded, yes, but since reconcile.Request doesn't include
+		// any kind of object meta, there's not much we can do.
+		Meta: &metav1.ObjectMeta{
```

Comment:
might be nice to abstract this somehow.
`r.reconcileNotifier.Notify()`, etc

Reply:
good point, done.